context/SocialMediaFeedContext.tsx

```diff
@@ -0,0 +1,76 @@
+import { createContext, useContext, useEffect, useState } from "react";
+import {
+  NEXT_PUBLIC_APP_URL,
+  NEXT_PUBLIC_SANITY_PROJECT_ID,
+} from "studio/config";
+
+interface SocialMediaFeedContextProps {
+  children: React.ReactNode;
+}
+
+const SocialMediaFeedContext = createContext(null); // pass default value
+
+export function useSocialMediaFeed() {
+  return useContext(SocialMediaFeedContext);
+}
+
+export function SocialMediaFeedContextProvider({
+  children,
+}: SocialMediaFeedContextProps) {
+  const initialState = {
+    itemId: "",
+    platform: "",
+    userName: "",
+    status: "loading",
+    media: [],
+  };
+
+  const URL =
+    "https://feat-socialmedia--dxpstudio-staging.netlify.app/api/social-accounts/media";
```

Comment:
Make sure this is configurable via env variable.

Reply:
Revision done. Thanks!